api/routes/auth-routes.js

```diff
@@ -0,0 +1,64 @@
+const router = require("express").Router();
+const passport = require("passport");
+const CLIENT_HOME_PAGE_URL = "http://localhost:3000/homepage";
+
+// when login is successful, retrieve user info
+router.get("/login/success", (req, res) => {
+  if (req.user) {
+    res.json({
+      isAuthenticated: true,
+      message: "user has successfully authenticated",
+      user: req.user,
+      cookies: req.cookies
+    });
+  }
+});
+
+const authCheck = (req, res, next) => {
+  if (!req.user) {
+    res.status(401).json({
```

Comment:
Just an observation. After logging in, it still comes to this part of the code when I call `http://localhost:9000/auth/checkStatus` which implies that req.user is null. But it shouldn't be if I have logged in successfully